### ISSUE TITLE
Add documentation for supported actions (remove_pkg, ...)

### ISSUE THREAD

**Issue by esciara** [pypiserver/pypiserver#267]
Hi,

I tried to find how to remove a package from `pypiserver`, but found it very hard to find clues. What saved me was #200, which just about mentioned a `remove_pkg` action. Then I found that this action was introduced in #31 .

I could not find any doc listing the available actions, whether in `pypiserver`'s doc or elsewhere. Even less about how to use them with a `curl` for install.

Could anyone point out some references?

**Comment by esciara** [comment]
As a note, the `curl` to remove a package is:
```
curl --form ":action=remove_pkg"  --form "name=<package_name>" --form "version=<version>"  <pypiserver_url>
```

**Comment by mplanchard** [comment]
Yeah, you're right. We don't currently document the functionality that's not exposed through twine/pip and the standard simple repository interface. I'll dive in and throw something into the readme at some point!

**Comment by esciara** [comment]
Thanks!

**Comment by ryan4yin** [comment]
@mplanchard does pypiserver provide any way to delete some releases of a package?
in order to save storage space, we are looking for ways to clean up historical data.

**Comment by mplanchard** [comment]
@ryan4yin yes, the curl command above should do it. There's a bug in the currently released version that will only result in one old package being removed, but that is fixed in master and will be included in the next release (see #273).

This ticket is high on my TODOs (and of course I'd accept PRs), which is to document that functionality, which is currently undocumented.

**Comment by AlexGustafsson** [comment]
Hi, @mplanchard. Sorry for sort of hijacking this issue. I depend on the feature fixed in https://github.com/pypiserver/pypiserver/pull/273. Any idea of when the next release containing the fix will happen?

**Comment by mplanchard** [comment]
Noting that we should include this documentation as part of the efforts in #367
